bootstrap.sh

```diff
@@ -36,10 +36,9 @@ case $2 in
     ;;
 esac
 
-# bootstrap pidev env
 if [[ ${devenv_name} =~ ubuntu1804/* ]]; then
   apt-get update -y \
-  && apt-get install -y curl git python-dev gcc python3-distutils;
+  && apt-get install -y python3-pip;
```

Comment:
use get-pip to perform initial install of pip, and pip itself to upgrade pip thereafter.